packages/cubejs-query-orchestrator/driver/BaseDriver.js

```diff
@@ -45,6 +45,14 @@ class BaseDriver {
     throw 'Not implemented';
   }
 
+  downloadQueryResults() {
+    throw 'Not implemented';
```

Comment:
It would be great to have `new Error('Not implemented')` instead of `'Not implemented'` here as it'll be easier to find this line when it's thrown.